src/helpers/set-utils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Returns the set difference of `a` minus `b`.
+ *
+ * @param a {Set|Array}
+ * @param b {Set|Array}
+ * @returns {Set}
+ */
+export function setDifference(a, b) {
```

Comment:
I think the names here would be more clear if they were reversed (diffSet and copySet)? Right now they read as mutators.